### PR TITLE
Functional Tests : Removed Product Page v1

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Functional Tests : Removed Product Page v1
| Type?             | bug fix
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | CI is :green_circle: 
| UI Tests          | https://github.com/Progi1984/ga.tests.ui.pr/actions/runs/6531460313 <br>https://github.com/Progi1984/ga.tests.ui.pr/actions/runs/6535206029
| Fixed issue or discussion?     | N/A
| Related PRs       | N/A
| Sponsor company   | @PrestaShopCorp

## Linked PRs : 
* https://github.com/PrestaShop/ga.tests.ui.pr/pull/57

### CODE REVIEW

**Review by nesrineabdmouleh — APPROVED** [review]

**Review by nicosomb — APPROVED** [review]
OK for the github action part.

File: tests/UI/pages/BO/catalog/products/add/stocksTab.ts [hunk 1]
```diff
@@ -225,7 +278,13 @@ class StocksTab extends BOBasePage {
    * @param label {string} Label to set when out of stock in the input
    */
   async setLabelWhenOutOfStock(page: Page, label: string): Promise<void> {
-    await this.setValue(page, `${this.productLabelAvailableLaterInput}_1`, label);
+    await this.waitForSelectorAndClick(page, this.productLabelAvailableLaterDropdown);
+    await this.waitForSelectorAndClick(page, this.productLabelAvailableLaterDropdownItem('en'));
+    await page.evaluate(
+      (selector: string) => (document.querySelector(selector) as HTMLElement).click(),
+      this.productLabelAvailableLaterDropdownItem('en'),
+    );
```
Comment (boubkerbribri) [inline tests/UI/pages/BO/catalog/products/add/stocksTab.ts:286]: same action done 2 times ?
```suggestion
    await page.locator(this.productLabelAvailableLaterDropdownItem('en'))
      .evaluate((el) => el.click());
```

**Review by boubkerbribri — COMMENTED** [review]
Little suggestion but LGTM.

File: tests/UI/pages/BO/catalog/products/add/stocksTab.ts (hunk 1 above)
Comment (boubkerbribri) [reply tests/UI/pages/BO/catalog/products/add/stocksTab.ts:286]: to be done on another PR

**Review by boubkerbribri — APPROVED** [review]

**Comment by boubkerbribri** [comment]
Thanks @Progi1984

File: tests/UI/pages/BO/catalog/products/add/stocksTab.ts (hunk 1 above)
Comment (Progi1984) [reply tests/UI/pages/BO/catalog/products/add/stocksTab.ts:286]: Fixed

**Pull request merged**